pkg/api-server/testdata/inspect_meshtrafficpermission.json

```diff
@@ -0,0 +1,19 @@
+{
+ "total": 1,
+ "items": [
+  {
+   "kind": "SidecarDataplane",
+   "dataplane": {
+    "mesh": "mesh-1",
+    "name": "backend-1"
+   },
+   "attachments": [
+    {
+     "type": "dataplane",
+     "name": "",
+     "service": ""
```

Comment:
Something is odd here

Reply:
This is how `dataplane` attachment is rendered https://github.com/kumahq/kuma/blob/master/pkg/api-server/testdata/inspect_traffic-trace.json#L23

Reply:
Actually are we just missing a `omitempty` on the type then?